core/utils/hashset/test.c

```diff
@@ -0,0 +1,223 @@
+#include <stdio.h>
+#include <assert.h>
+#include <string.h>
+#include "hashset/hashset.h"
+#include "hashset/hashset_itr.h"
+
+static void trivial(void)
```

Comment:
Is this code run in CI?

Reply:
This code is not run in CI. I'm not sure how to do that.